home-manager/home-manager

```diff
@@ -920,7 +924,7 @@ function doHelp() {
     echo
     echo "  help         Print this help"
     echo
-    echo "  edit         Open the home configuration in \$EDITOR"
+    echo "  edit         Open the home configuration in \$VISUAL or \$EDITOR"
```

Comment:
This would need a similar update in `docs/home-manager.1`.

Reply:
Done 🙂